File: SPCode/CS/Data/Support/SPPayModeCategories.cs
using System.Text.RegularExpressions;
using System.Diagnostics;
using System;
using System.Drawing.Drawing2D;
using System.Windows.Forms;
using System.Collections;
using System.Drawing;
using Microsoft.VisualBasic;
using System.Data.SqlClient;
using System.Data;
using System.Collections.Generic;
using WeifenLuo.WinFormsUI;
using Microsoft.Win32;
using WeifenLuo;


namespace SoftLogik.Win
{
	public class SPPayModeCategories
	{
		
		public static SPPayModeItem[] DefaulList
		{
			get
			{
				SPPayModeItem[] _DefaultList = {new SPPayModeItem("Cash", "CASH"), new SPPayModeItem("Credit", "CREDIT"), new SPPayModeItem("Cheque", "CHEQUE"), new SPPayModeItem("On Account", "ACCOUNT"), new SPPayModeItem("Voucher", "VOUCHER"), new SPPayModeItem("Other", "OTHER")};
				return _DefaultList;
			}
		}
		
		
	}
	
	
	public class SPPayModeItem
	{
		
		
		private string _Name;
		public string Name
		{
			get
			{
				return _Name;
			}
		}
		
		private string _Category;
		public string Category
		{
			get
			{
				return _Category;
			}
		}
		
		internal SPPayModeItem(string Name, string Category)
		{
			this._Name = Name;
			this._Category = Category;
		}
	}
	
	
}

File: SPCode/Data/Support/SPPayModeCategories.vb

Public Class SPPayModeCategories
    Public Shared ReadOnly Property DefaulList() As SPPayModeItem()
        Get
            Dim _DefaultList() As SPPayModeItem = {New SPPayModeItem("Cash", "CASH"), _
                                                    New SPPayModeItem("Credit", "CREDIT"), _
                                                    New SPPayModeItem("Cheque", "CHEQUE"), _
                                                    New SPPayModeItem("On Account", "ACCOUNT"), _
                                                    New SPPayModeItem("Voucher", "VOUCHER"), _
                                                    New SPPayModeItem("Other", "OTHER")}
            Return _DefaultList
        End Get
    End Property


End Class


Public Class SPPayModeItem

    Private _Name As String
    Public ReadOnly Property Name() As String
        Get
            Return _Name
        End Get
    End Property

    Private _Category As String
    Public ReadOnly Property Category() As String
        Get
            Return _Category
        End Get
    End Property

    Friend Sub New(ByVal Name As String, ByVal Category As String)
        Me._Name = Name
        Me._Category = Category
    End Sub
End Class

